### ISSUE TITLE
Publish project in NPM

### ISSUE THREAD

**Issue by EdwinBetanc0urt** [PanJiaChen/vue-element-admin#1467]
Has not been thought of publishing this project in NPM? to make direct installations with NPM without having to download from github

**Comment by PanJiaChen** [comment]
I don't think this is a good idea. This project is a template, not a library. How do you use it after downloading from npm?

**Comment by EdwinBetanc0urt** [comment]
It could be similar to the React create-react-app or the vue init webpack my-project, so that the project is created with instructions like vue-element-admin init or create my-app

**Comment by justforuse** [comment]
you can use yeoman to generate your basic project code
